src/components/SidePanel/NavGroup/HighlightNavGroup/index.tsx

```diff
@@ -28,7 +29,7 @@ const HighlightNavGroup: React.FC = () => {
       items={highlights}
       navGroupTitle="Highlighted Terms"
     >
-      {highlights.map((highlight) => (
+      {highlights.map((highlight, index) => (
```

Comment:
I'm somewhat wondering if we should do something more robust than an array. I was thinking about in the future, if we would want to allow users to change the color and provide some colors to choose from.

<img width="499" alt="Screen Shot 2023-02-07 at 3 46 49 PM" src="https://user-images.githubusercontent.com/47064971/217364870-ee053aaf-303f-4789-a93a-a49a0fe5e1df.png">

Reply:
As awesome as that looks I would push back against it at least for this first iteration since that would dramatically increase implementation complexity for little benefit IMO. Right now the colors are independent of app state doing it that way would require us to keep track of what colors are associated with what highlights.

Reply:
Okay, that makes sense to me to not to support it for that use case at the current time 👍

I wanted to point out one other scenario where this might come into question, which is when we delete highlights. Deleting highlights will change the colors in view, which I'm not sure how users will feel about. (This could be more of a product/design question, but wanted to flag.)

https://user-images.githubusercontent.com/47064971/218574955-e68a2b49-ca2d-46e0-8d81-ec6f91e71c3c.mov

Reply:
I think this is an ok compromise especially given the low usage of this feature.